src/main/resources/application.properties

```diff
@@ -23,4 +23,10 @@ scheduler.cdc.queue.ttlSeconds=\${SCHEDULER_CDC_QUEUE_TTL_SECONDS}
 scheduler.cdc.queue.visibilityTimeoutWalletCdc=\${SCHEDULER_CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS}
 # Retry policy for sending message
 scheduler.cdc.retry-send.maxAttempts=\${SCHEDULER_CDC_SEND_RETRY_MAX_ATTEMPTS}
-scheduler.cdc.retry-send.intervalInMs=\${SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS}
\ No newline at end of file
+scheduler.cdc.retry-send.intervalInMs=\${SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS}
+# CDC resume policy
+cdc.redis-resume.keyspace=\${CDC_REDIS_RESUME_KEYSPACE}
+cdc.redis-resume.target=\${CDC_REDIS_RESUME_TARGET}
+cdc.redis-resume.fallbackInMin=\${CDC_REDIS_RESUME_FALLBACK_IN_MIN}
+cdc.redis-resume.ttlInMin=\${CDC_REDIS_RESUME_TTL_IN_MIN}
+cdc.resume.saveInterval=\${CDC_RESUME_SAVE_INTERVAL}
```

Comment:
Please remove save interval, the job will be responsible to handle the configuration for saving checkpoints.
Please consider using the prefix scheduler. for these properties and removing cdc.redis-resume.target, the job id will be passed to service's method as an input parameter

Reply:
done it here 1acaceec11c78fc66778f1ef4f816f0dc0e1a1a7